latency-log-cache/go.mod

```diff
@@ -1,13 +1,22 @@
 module github.com/cloudfoundry-incubator/loggregator-tools/latency
 
+go 1.17
+
 require (
-	code.cloudfoundry.org/go-loggregator v7.6.0+incompatible
-	code.cloudfoundry.org/log-cache v0.0.0-00010101000000-000000000000
-	github.com/cloudfoundry/noaa v2.1.0+incompatible
-	github.com/cloudfoundry/sonde-go v0.0.0-20171206171820-b33733203bb4
-	github.com/elazarl/goproxy v0.0.0-20181003060214-f58a169a71a5 // indirect
-	github.com/gorilla/websocket v1.4.0 // indirect
+	code.cloudfoundry.org/go-log-cache v1.0.1-0.20211123204853-93b7f0836b81
+	code.cloudfoundry.org/go-loggregator/v8 v8.0.2-0.20200722201844-b5130958b65d
 	github.com/nu7hatch/gouuid v0.0.0-20131221200532-179d4d0c4d8d
 )
 
-replace code.cloudfoundry.org/log-cache => code.cloudfoundry.org/log-cache-release/src v2.0.2-0.20190801153242-56147f52a11c+incompatible
+require (
+	code.cloudfoundry.org/rfc5424 v0.0.0-20180905210152-236a6d29298a // indirect
+	github.com/blang/semver v3.5.1+incompatible // indirect
+	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.6.0 // indirect
+	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4 // indirect
+	golang.org/x/sys v0.0.0-20210510120138-977fb7262007 // indirect
+	golang.org/x/text v0.3.5 // indirect
+	google.golang.org/genproto v0.0.0-20210903162649-d08c68adba83 // indirect
+	google.golang.org/grpc v1.40.0 // indirect
+	google.golang.org/protobuf v1.27.1 // indirect
```

Comment:
A bit weird to bring in both protobuf packages.. It's indirect though so can't really be helped I guess. Seems like go-loggregator uses github.com/golang/protobuf and go-log-cache uses google.golang.org/protobuf.

Reply:
going to ignore this duplication for now since this isn't production code.

Reply:
This was a while ago. I've realized since that it's necessary.